test/integration/test_datatype_upload_irods.py

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
```suggestion
import os
```
Got a bit carried away here :)

Reply:
:)